config/initializers/simple_form.rb

```diff
@@ -62,7 +62,7 @@
   # Defaults to :nested for bootstrap config.
   #   inline: input + label
   #   nested: label > input
-  config.boolean_style = :nested
+  config.boolean_style = :inline
```

Comment:
@tahb I reverted this on my local, which was causing the checkbox not to appear.